MobileWeather/app/src/main/java/com/ford/mobileweather/activity/MainActivity.java

```diff
@@ -1,7 +1,6 @@
 package com.ford.mobileweather.activity;
```

Comment:
The onCreate method should include the call to ` SmartDeviceLinkReceiver.queryForConnectedService(this);`